tools/travis/linux_release/after_success.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/bash
 #set -ev
-
 if [[ "${TRAVIS_PULL_REQUEST}" == "false" ]]; then
     echo -e "Packaging binaries and libs"
 
-    # === Copy Libs ===
-	QT_LIB_DIR=/opt/qt510/lib
-    QT_LIBS=(libQt5Charts libQt5Concurrent libQt5Core libQt5Gui libQt5Network libQt5PrintSupport libQt5Qml libQt5Svg libQt5Test libQt5Widgets libQt5Xml libQt53DCore libQt53DExtras libQt53DInput libQt53DLogic libQt53DRender )
-    n_elements=${#QT_LIBS[@]}
-    for ((i = 0; i < n_elements; i++)); do
-        libpath="$QT_LIB_DIR/${QT_LIBS[i]}.*"
-        cp $libpath ./lib
-    done
+    # === Linux Deploy Qt ===
+	#sourcing qt510 again, this is for linuxdeployqt; verified redundant, so commented out. 
+	#source /opt/qt510/bin/qt510-env.sh
+
+	#setting linuxdeployqt to variable
+	linuxdeployqt=linuxdeployqt-continuous-x86_64.AppImage
+
+	#archive file name created
+	archive_name="mne-cpp-linux-x86_64-$TRAVIS_BRANCH.tar.gz"
+
+	#copying built data to folder for easy packaging 	
+	cp -r ./bin ./lib mne-cpp/
+
+	#dropping into folder to easily package all results from linuxdeployqt
+	cd mne-cpp
+
+	#linuxdeployqt uses mne_scan binary to resolve dependencies in current directory. 
+	../$linuxdeployqt bin/mne_scan
```

Comment:
Does linuxdeployqt copy the dependencies into the current working directory? I always thought it copies them relative to the binary, in our case /bin/mne_scan . If this is true we would miss out on the libs copied to /lib and /plugins. How about moving the `cp -r ./bin ./lib mne-cpp/` statement after running linuxdeployqt and also add `./plugins` to it?

Reply:
In my tests with linuxdeployqt the working directory is the location everything is built in. I can check if there is an option to assign another target directory.  In the prior tests with the build linuxdeployqt creates a plugins folder in the working directory.

Reply:
hmm. I just tried it on my CentOS 7 and it did not work for me. Do you mind double checking on your test env again? Also, `../$linuxdeployqt bin/mne_scan` did not work for me. I used `../$linuxdeployqt ../bin/mne_scan` instead.

Reply:
Tested again with ubuntu 14.04.6 LTS. It looks like your active directory wasn't inside the created mne-cpp folder. My check built correctly with a folder containing the linuxdeployqt output.  If you have a plugins directory in the main build directory you executed linuxdeployqt outside of the folder created to for linuxdeployqt to create all needed files. 

isk@isk-VirtualBox:~/imsorryk-git/mne-cpp/mne-cpp$ ls -ls
total 20
4 drwxr-xr-x 7 root root 4096 Jun  4 21:16 bin
4 drwxr-xr-x 2 root root 4096 Jun  4 21:15 lib
4 drwxr-xr-x 8 root root 4096 Jun  4 21:16 plugins
4 drwxr-xr-x 3 root root 4096 Jun  4 21:15 share
4 drwxr-xr-x 2 root root 4096 Jun  4 21:16 translations

Reply:
Yes, you were right. Mistake on my side. Sry for that.